digesta-bot/action.yml

```diff
@@ -44,6 +47,7 @@ runs:
 
     - shell: bash
       run: |
+        set +e
```

Comment:
nit:
```suggestion
        set -o errexit
```
easier to read

Reply:
here I want to disable, by default is enabled

Reply:
ah, so `set + errexit`